### ISSUE TITLE
Supporting subdirectories via reverse proxy

### ISSUE THREAD

**Issue by Insighter2k** [wger-project/wger#1403]
I am running different apps behind a reverse proxy. Using my current domain with different subdirectories. Unfortunately, I haven't found any setting / configuration for configuring this app for a subdomain. In the readme for the deployment, only a domain or subdomain is supported, but no subdirectory.  Are there any plans to add this feature? I do not want to add the hassle of adding another subdomain and certification process for just this app. Thanks for the work so far :-)

Kind regards

**Comment by rolandgeider** [comment]
Hi! While there isn't a technical reason why this couldn't work, there are probably a few places where we assume that it is running directly under / so some things would probably break

So basically we'd need to sit down, test it and fix everything step by step

**Comment by rolandgeider** [comment]
exactly, and I would guess there shouldn't be that many places to fix (famous last words 😅 )
